docs/en/cloud/reference/changelog.md

```diff
@@ -6,6 +6,27 @@ title: Cloud Changelog
 
 In addition to this ClickHouse Cloud changelog, please see the [Cloud Compatibility](/docs/en/cloud/reference/cloud-compatibility.md) page.
 
+## May 4, 2023
+
+This release brings a new heatmap chart type, improves billing usage page, and improves service startup time.
+
+### Console changes
+- Added heatmap chart type to SQL console
+- Improved billing usage page to show credits consumed within each billing dimension
+
+### Integrations changes
+- Kafka connector: Added retry mechanism for transient connection errors
+- Python client: Added max_connection_age setting to ensure that HTTP connections are not reused forever. It can help with certain load balancing issues
```

Comment:
```suggestion
- Python client: Added max_connection_age setting to ensure that HTTP connections are not reused forever. This can help with certain load-balancing issues
```